docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -1,38 +1,31 @@
 = Migrating Ceph Manager daemons to {Ceph} nodes
 
-The following section describes how to move Ceph Manager daemons from the
-{rhos_prev_long} Controller nodes to a set of target nodes. Target nodes might
-be pre-existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is
-deployed by {OpenStackPreviousInstaller} with an HCI topology.
-This procedure assumes that Cephadm and the {Ceph} Orchestrator are the tools
-that drive the Ceph Manager migration. As is done with the other Ceph daemons
-(MDS, Monitoring, and RGW), the procedure uses the Ceph spec to modify the
-placement and reschedule the daemons. Ceph Manager is run in an active/passive
-fashion, and it also provides many modules, including the Ceph orchestrator.
+You must migrate your Ceph Manager daemons from the {rhos_prev_long} ({OpenStackShort}) Controller nodes to a set of target nodes. Target nodes are either existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is deployed by {OpenStackPreviousInstaller} with a Hyperconverged Infrastructure (HCI) topology.
 
+The following procedure uses `cephadm` and the Ceph Orchestrator to drive the Ceph Manager migration, and the Ceph spec to modify the placement and reschedule the Ceph Manager daemons. Ceph Manager is run in an active/passive state, and it also provides many modules, including the Ceph Orchestrator.
```

Comment:
Please review this paragraph . And does “it” refer to Ceph Manager in this sentence: "...and it also provides many modules,...".